docker/test/integration/runner/Dockerfile

```diff
@@ -94,6 +94,10 @@ RUN python3 -m pip install \
     azure-storage-blob \
     meilisearch==0.18.3 
 
+ARG TARGETARCH
+RUN curl -fsSL https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-linux-${TARGETARCH} > /usr/bin/mkcert \
+    && chmod +x /usr/bin/mkcert
```

Comment:
It it possible to reuse `generate_certs.sh` from other tests?

Reply:
Will check it out

Reply:
Can't find `generate_certs.sh` anywhere

Reply:
https://github.com/ClickHouse/ClickHouse/blob/12291d95ddc9a79357cc9be7dd210870628f5803/tests/integration/test_ssl_cert_authentication/certs/generate_certs.sh

Reply:
Ah, it's a client cert for 10 years. 
Server cert length is 1 year only. Which means that these static ones will need to be regenerated. I suppose creating them dynamically with `mkcert` is more fail-proof.

Reply:
> Server cert length is 1 year only.

Why? I didn't know we have a check for this.

Reply:
Nobody has check for this, it's openssl-enforced, and the implementation does change too.
For example current TLS implementation is hardcoded whcih is a bad practice, because updating openssl on the host/image will not update clickhouse binary

Reply:
If next year openssl will adopt something like this: https://techbeacon.com/security/google-apple-mozilla-enforce-1-year-max-security-certifications the tests will break. If we generate for one year, they will break in one year.
Etc.
I think re-generating them on each run is not a real problem.

Reply:
It's totally fine to re-generate certificates on each test run, I mean that we probably don't need third party dependency for this (especially if it can be done in a few lines in bash). Also tests will break (in N years) if `FiloSottile` decide to remove old releases, for example...

Reply:
That's a valid point, but we could have started with that one.
It's more than a couple of bash lines though, as it has support for various cert stores (clickhouse on macos, ...ahem)
But I can rewrite it into openssl commands if that's ok?

Reply:
Okay, removed all changes to the dockerfile for runner